golem-cluster.override.2-image.yaml

```diff
@@ -2,4 +2,4 @@ provider:
   parameters:
     node_config:
       demand:
-        image_tag: "blueshade/ray-on-golem:0.8.0-py3.10.13-ray2.9.2"
+        image_tag: "golem/ray-on-golem:0.8.0-py3.10.13-ray2.9.2"
```

Comment:
I think this is important for pre release versions to be not from the `golem` user registery.